README.md

```diff
@@ -82,7 +82,7 @@ Want to log with Aardvark but don’t want to use Aardvark’s bug reporting too
 ## Requirements
 
 * Xcode 8.0 or later
```

Comment:
Let's go ahead and bump the supported Xcode version to 11 alongside this. I don't think old Xcode support is buying us much at this point, but it makes our test matrix more complicated.

Reply:
Bumped this version in the README.md, and .travis.yml. Not sure if there's anything else to bump?

Reply:
I think that should cover it. 👍  Thanks!